bridgestyle/sld/fromgeostyler.py

```diff
@@ -315,22 +315,22 @@ def _lineSymbolizer(sl, graphicStrokeLayer=0):
         _addCssParameter(stroke, "stroke-opacity", opacity)
         _addCssParameter(stroke, "stroke-linejoin", join)
         _addCssParameter(stroke, "stroke-linecap", cap)
-        if dasharray is not None:
-            if cap != "butt":
-                try:
-                    EXTRA_GAP = 2 * width
-                    tokens = [
-                        int(v) + EXTRA_GAP if i % 2 else int(v)
-                        for i, v in enumerate(dasharray.split(" "))
-                    ]
-                except:  # in case width is not a number, but an expression
-                    GAP_FACTOR = 2
-                    tokens = [
-                        int(v) * GAP_FACTOR if i % 2 else int(v)
-                        for i, v in enumerate(dasharray.split(" "))
-                    ]
-                dasharray = " ".join([str(v) for v in tokens])
-            _addCssParameter(stroke, "stroke-dasharray", dasharray)
+    if dasharray is not None:
+        if cap != "butt":
+            try:
+                EXTRA_GAP = 2 * width
+                tokens = [
+                    int(v) + EXTRA_GAP if i % 2 else int(v)
+                    for i, v in enumerate(dasharray.split(" "))
+                ]
+            except:  # in case width is not a number, but an expression
```

Comment:
Can we check that with a simple if ?

Reply:
Same as earlier comment: it's in the main code and we don't have any unit test, so I prefer to leave it as is (I did not write the code, I only changed the indentation because it seemed off)